File: MODUL 5 PEMROGRAMAN 1/PRAK505-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>

void Biodata(int tahunlahir, char A[],char B[]){
int tahun_sekarang = 2020;
    printf("Perkenalkan Nama Saya: %s\n", A);
    printf("Umur Saya: %d\n", tahun_sekarang - tahunlahir);
    printf("Saya Adalah Angkatan: %d\n", tahun_sekarang);
    printf("Asal Saya Dari: %s\n", B);
}

int main() {
    int tahunLahir;
    char A[20], B[15];
    scanf("%d",&tahunLahir);
    scanf("%s",&A);
    scanf("%s",&B);
    Biodata(tahunLahir, A, B);
    return 0;
}

File: MODUL 5 PEMROGRAMAN 1/PRAK505-2310817120002-SITI RATNA DWINTA SARI.py
def biodata(tahun_lahir, A, B ):
    tahun_sekarang = 2020
    print("Perkenalkan Nama Saya:", A)
    print("Umur Saya:", tahun_sekarang - tahun_lahir)
    print("Saya Adalah Angkatan:", tahun_sekarang)
    print("Asal Saya Dari:", B)

tahun_lahir = int(input())
nama = input()
asal = input()
biodata(tahun_lahir, nama, asal)
